src/main/java/core/basesyntax/Truck.java

```diff
@@ -0,0 +1,14 @@
+package core.basesyntax;
+
+//new class Truck
+public class Truck extends Machine {
+    @Override
+    public void doWork() {
+        System.out.println("Truck started its work");
+    }
+
+    @Override
+    public void stopWork() {
+        System.out.println("Truck stopped working");
```

Comment:
Similarly, the message in the 'stopWork' method should be more informative. Consider changing it to 'Truck has stopped working'.